src/test/unit_tests/jest_tests/test_nc_account_cli.test.js

```diff
@@ -2218,6 +2245,35 @@ async function exec_manage_cli_add_empty_option(command, option) {
     return res;
 }
 
+/**
+ * exec_manage_cli_add_leading_space_option modifies the command by appending a flag with a value(quoted or non-quoted) that includes a leading space to test CLI parsing behavior
+ * @param {string} command
+ * @param {string} option
+ * @param {string} value
+ * @param {string} quoted
```

Comment:
Could you add examples in the function description for the different cases?

Reply:
sure, thanks